ome/rOMERO-gateway
# update dependencies

Fix issue noticed when review #87 
https://github.com/ome/rOMERO-gateway/pull/87#issuecomment-591011342

To test:
 * Go to ``jupyter`` folder
 * Run ``docker build -t romero .``
 * This should now work
cc @dominikl
